tikv/region.go

```diff
@@ -313,6 +316,25 @@ func (rm *RaftRegionManager) OnRegionConfChange(ctx *raftstore.PeerEventContext,
 	region.updateRegionEpoch(epoch)
 }
 
+func (rm *RaftRegionManager) OnRoleChange(ctx *raftstore.PeerEventContext, newState raft.StateType) {
+	rm.mu.RLock()
+	region, ok := rm.regions[ctx.RegionId]
+	rm.mu.RUnlock()
+	if ok {
+		if bytes.Compare(region.startKey, []byte{}) == 0 {
+			newRole := Follower
+			if newState == raft.StateLeader {
+				newRole = Leader
+			}
+			log.Infof("[ray] first region role change newRole=%v", newRole)
```

Comment:
What does `ray` mean?

Reply:
temporary debug log, will be removed later.